mission-control/docs/playbooks/actions/exec.md

```diff
@@ -48,12 +58,44 @@ Exec Check executes a command or script file on the target host. The type of scr
 | `clientSecret`   | Client Secret   | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
 | `tenantID`       | Tenant ID       | `string`                                                                      |          |
 
-## Templating
+## Artifact
+
+| Field  | Description   | Type     | Required |
+| ------ | ------------- | -------- | -------- |
+| `path` | Path or glob. | `string` | `true`   |
+
+[Read more ...](../concepts/artifacts.md)
+
+## Git Checkout
+
+For authentication, either provide the connection name or the basic auth or the certificate.
+
+| Field         | Description                                                                                                                                                    | Type                                                                          | Required |
+| ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------- | -------- |
+| `url`         | Git repository URL.                                                                                                                                            | `string`                                                                      | `true`   |
+| `connection`  | Specify the connection name to use for git authentication (if required)                                                                                        | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `username`    | Git auth username.                                                                                                                                             | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `password`    | Git auth password.                                                                                                                                             | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `certificate` | Git auth certificate.                                                                                                                                          | [`types.EnvVar`](https://pkg.go.dev/github.com/flanksource/duty/types#EnvVar) |          |
+| `destination` | Destination is the full path to where the contents of the URL should be downloaded to. If left empty, the sha256 hash of the URL will be used as the dir name. | `string`                                                                      |          |
 
-The scripts you provide will be templated using the following variables:
+## Examples
```

Comment:
Can we move the example to the top and give it a filename like we do for all the canary checks